css/main.css

```diff
@@ -200,9 +193,74 @@ body {
 
 /* ––––––––– END FUNDRAISING ––––––––– */
 
+/* ––––––––– VISION ––––––––– */
+
+.vision {
+  background: url("/images/vision-bg.png") no-repeat center #FBFAFF;
+  background-size: cover;
+}
+
+.vision__inner {
+  padding: 200px 130px;
+}
+
+.vision-title {
+  margin-top: 6px;
+}
+
+.vision-text {
+  font-size: 18px;
+  color: var(--c-main-body);
+  line-height: 28px;
+  margin-top: 12px;
+}
+
+.vision-item__text {
```

Comment:
It seems like we can delete this class because what `display: flex; flex-direction: column;` does is put everything in a column, which is the default behavior without `display: flex`